packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -34,6 +43,39 @@ describe('displayStartTime', () => {
     expect(moment.utc).toHaveBeenCalledWith(startDate, null, expect.any(String))
   })
 
+  it.each([[50], [10000], [0]])(
+    'expected text replaces the date string if start after payment is selected using correct constant',
+    async startAfterMinutes => {
+      constant.START_AFTER_PAYMENT_MINUTES = startAfterMinutes
+      const startDate = '2021-01-01T00:00:00.000Z'
+      const mssgs = getSampleRequest().i18n.getCatalog()
+      const expectedReturnText = constant.START_AFTER_PAYMENT_MINUTES + mssgs.licence_summary_minutes_after_payment
+      moment.utc.mockImplementation(() => ({
+        tz: () => ({
+          locale: jest.fn(() => ({ format: () => '' })),
+          format: () => {}
+        })
+      }))
+      const startAfterPaymentMinutes = displayStartTime(getSampleRequest(), { startDate, licenceToStart: 'after-payment' })
+      expect(startAfterPaymentMinutes).toEqual(expectedReturnText)
```

Comment:
This, and the following test, are both incredibly complicated, especially considering what you're testing. 
1) Why are you adding a new mock for moment, that is identical to the mock you've created on lines 16-18?
2) Why are you getting a sample request twice? Create a `mockRequest` at the start of the test